osg-system-profiler

```diff
@@ -205,6 +205,13 @@ if $rpm_install; then
     dump_files_in_dir "/etc/osg/config.d"
 fi
 
+echo "***** Xrootd information:"
+dump_files_in_dir "/etc/xrootd"
+
+echo "***** Running: ps aux | grep xrootd"
+ps aux | grep "^xrootd"
```

Comment:
This command gives you all the processes owned by the xrootd user. Is that what you want? If that's true, you should change the echo line above to say so. (Also you can avoid the "grep" call by doing `ps -u xrootd u`.)

Reply:
Thanks for the suggestion @matyasselmeci I've added a new commit to the PR